NCATS_Data_Dictionary.md

```diff
@@ -3,7 +3,7 @@
 ## March 28, 2019 ##
 
 This document provides a data dictionary for the data stored in the
-following NoSQL MongoDB databases:
+following NoSQL MongoDB databases: 
```

Comment:
Your rampaging "search and replace all" is putting spaces after colons where no spaces should be, like here.